tests/integration/components/bs-form/element/label-test.js

```diff
@@ -42,6 +42,14 @@ module('Integration | Component | bs form/element/label', function (hooks) {
     assert.dom('label').hasClass('form-check-label', 'component has form-check-label class');
   });
 
+  testBS4('component has custom-control-label class when using control type switch', async function (assert) {
+    await render(hbs`<BsForm::Element::Label @controlType="switch" />`);
```

Comment:
As above I would prefer to only use public API in tests:

```suggestion
    await render(hbs`
      <BsForm as |form|>
        <form.element @controlType="switch" />
      </BsForm>
    `);
```